news/2023/2023-01-31-skinning-poll-2023.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "osu! 2023 Skinning Poll"
+date: 2023-01-31 17:00:00 +0000
+---
+
+Are you an avid skin creator who makes skins from scratch, or someone who makes remixed skins for fun? Maybe you are just a person who likes collecting skins and exploring what the community has to offer? Either way, we want your feedback on everything skinning related!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+With skinning continuously evolving, we created a survey through which the community can express their opinion on the current state of the skinning scene and help us better understand the community's expectations and needs.
+
+The link below will lead you to the survey page, while most questions are not mandatory to answer, the more feedback we receive, the easier it will be for us to make changes you want to see happen come to life.
```

Comment:
```suggestion
The link below will lead you to the survey page. While most questions are not mandatory to answer, the more feedback we receive, the easier it will be for us to make changes you want to see come to life.
```